lib/location/Kconfig

```
@@ -36,6 +36,10 @@ config LOCATION_METHODS_LIST_SIZE
 	help
 	  Maximum number of location methods within location_config structure.
 
+config LOCATION_WORKER_STACK_SIZE
+	int "Stack size for the thread acquiring a location"
```

Maybe the naming could also be changed to `LOCATION_WORKQUEUE_STACK_SIZE`? It would be aligned for example with Zephyr `SYSTEM_WORKQUEUE_STACK_SIZE`.
```suggestion
	int "Stack size for the library work queue"
```

---

A good point, changed.